easybuild/tools/systemtools.py

```diff
@@ -133,76 +103,107 @@ def get_core_count():
 
 
 def get_cpu_vendor():
-    """Try to detect the cpu identifier
+    """
+    Try to detect the CPU vendor
 
-    will return INTEL, ARM or AMD constant
+    @return: INTEL, ARM or AMD constant
```

Comment:
make a reference to the `VENDORS` constant instead trying to keep another list here